lib/EnsEMBL/REST/Model/LDFeatureContainer.pm

```diff
@@ -64,27 +63,23 @@ sub fetch_LDFeatureContainer_variation_name {
   Catalyst::Exception->throw("Could not retrieve a variation feature.") if (scalar @vfs == 0);
   my $vf = $vfs[0];
 
-  my $population_name = $c->request->param('population_name');
-  if ($population_name) {
-    my $pa = $c->model('Registry')->get_adaptor($species, 'Variation', 'Population');     
-    my $population = $pa->fetch_by_name($population_name);
-    if (!$population) {
-      Catalyst::Exception->throw("Could not fetch population object for population name: $population_name");
-    }
-    my $ldfc;
-    try {
-      $ldfc = $ldfca->fetch_by_VariationFeature($vf, $population);
-    } catch {
-      Catalyst::Exception->throw("Something went wrong while fetching from LDFeatureContainerAdaptor");
-    };
-    return $self->to_array($ldfc)
+  my $pa = $c->model('Registry')->get_adaptor($species, 'Variation', 'Population');     
+  my $population = $pa->fetch_by_name($population_name);
```

Comment:
The PopulationAdaptor returns undef if the population isn't stored in our database.
population_name is now required for this endpoint. I already check in the controller if population_name is given and throw an exception if not.